### DISCUSSION TITLE
MeshCentral Guest Sharing Questions

### DISCUSSION THREAD

**Discussion by SmoothPlay** [Ylianst/MeshCentral#5026] [Q&A]
Hello,

Before anything else, I want to thank the team behind Meshcentral one more time for how amazing this Software is.

Now for some questions:

1- Is it possible to Disable the Toolbar for Guest Sharing?

2- Is it possible to Disable User Consent / Notifying when a Guest connects?

3- Is it possible to  completely Disable the Keyboard Shortcuts for Guests?

4- Is it possible to immediately start the connection as soon as we enter the Guest's link?

5- When accessing guest shared iframe link on mobile, the browser redirects to the original link, is it possible to change this? ( apparently on PC the iframe is okay )

6- Is it possible to start the Guest session in full screen without the bars on top and on the bottom?

Basicly I would like the guest to only control the mouse in the most user friendly, secure and clean way.

I might be asking too much :P

I don't mind having a dedicated server behaving like this, let me know if it is possible even if internal server files must be changed.

**Comment by SomeGuru** [comment]
Honestly all your requests here might be manageable with you running a custom ElectronJS application that you setup the parameters. Reason being is that it allows you to set what you want the kiosk user to have access too... it worked when me and my team played with it a few years ago before covid-19 lockdowns and it was used for a trade show to see if users of the public could get around the tool itself... They said it was very locked down, but was usable to control a subset of things on the remote end.

[ElectronJS App Building +Browser] (https://electronjs.org/docs/latest/tutorial/tutorial-first-app)

-SomeGuru
